plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteClient.java

```diff
@@ -366,13 +367,18 @@ public JdbcOutputTableHandle beginCreateTable(ConnectorSession session, Connecto
 
         int expectedSize = tableMetadata.getColumns().size();
         ImmutableList.Builder<String> columns = ImmutableList.builderWithExpectedSize(expectedSize);
-        ImmutableList.Builder<String> columnNames = ImmutableList.builderWithExpectedSize(expectedSize);
+        ImmutableList.Builder<String> columnNamesBuilder = ImmutableList.builderWithExpectedSize(expectedSize);
         ImmutableList.Builder<Type> columnTypes = ImmutableList.builderWithExpectedSize(expectedSize);
         for (ColumnMetadata columnMetadata : tableMetadata.getColumns()) {
             columns.add(getColumnDefinitionSql(session, columnMetadata, columnMetadata.getName()));
-            columnNames.add(columnMetadata.getName());
+            columnNamesBuilder.add(columnMetadata.getName());
             columnTypes.add(columnMetadata.getType());
         }
+
+        List<String> columnNames = columnNamesBuilder.build();
+
+        checkPrimaryKeyExistsForBeginCreateTable(columnNames, IgniteTableProperties.getPrimaryKey(tableMetadata.getProperties()));
+
```

Comment:
You can check like this:
```java
        List<String> columnNames = columnNamesBuilder.build();
        List<String> primaryKeys = IgniteTableProperties.getPrimaryKey(tableMetadata.getProperties());
        for (String primaryKey : primaryKeys) {
            if (!columnNames.contains(primaryKey.toLowerCase(ENGLISH))) {
                throw new TrinoException(INVALID_TABLE_PROPERTY, format("Column '%s' specified in property '%s' doesn't exist in table", primaryKey, PRIMARY_KEY_PROPERTY));
            }
        }

        String sql = buildCreateSql(schemaTableName, columns.build(), primaryKeys);
...
                    primaryKeys.isEmpty() ? Optional.of(IGNITE_DUMMY_ID) : Optional.empty());
...
    private String buildCreateSql(SchemaTableName schemaTableName, List<String> columns, List<String> primaryKeys)
```

Reply:
done

Reply:
@ebyhr Have added the check in the above way.

Where do we want the ternary expression check?
```
 primaryKeys.isEmpty() ? Optional.of(IGNITE_DUMMY_ID) : Optional.empty());
```

Reply:
You can fix L399.

Reply:
done